cmd/kubeadm/app/constants/constants.go

```diff
@@ -370,6 +370,16 @@ const (
 	// May be overridden by a flag at startup.
 	KubeControllerManagerPort = 10257
 
+	// EtcdAdvertiseClientUrlsAnnotationKey is the annotation key on every etcd pod, describing the
+	// advertise client URL
```

Comment:
`URLs` plural?

Reply:
or rather the key value should be `-...client-url`

Reply:
and if so we have to adjust this in the release note:

> kubeadm: The ClusterStatus struct present in the kubeadm-config ConfigMap <snip> `kubeadm.kubernetes.io/etcd.advertise-client-urls` ...

Reply:
Being the etcd flag `--advertise-client-urls` I'm ok with `EtcdAdvertiseClientUrlsAnnotationKey`